.github/workflows/swagger.yml

```diff
@@ -0,0 +1,47 @@
+name: Open API Specification and Documentation
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - 'master'

```

Comment:
The workflow is not being triggered yet. Let's force more the conditions. Remove all the `push` and `pull_request` segments and set it to:
```yaml
on: [push, pull_request]
```

as you can see in this commit/PR/line:
https://github.com/layer5io/meshery/pull/3038/files#diff-42244398366244a59eb9bb95d4d4f8f8657297b9e8e5bf3a815f3e8cca374ce1R3